content/en/docs/ops/ambient/install/helm-installation/snips.sh

```diff
@@ -42,7 +42,7 @@ helm install ztunnel istio/ztunnel -n istio-system
 }
 
 snip_install_ingress() {
-helm install istio-ingress istio/gateway -n istio-ingress --wait --create-namespace
```

Comment:
Can we just consistently use `--wait` everywhere?

It's useful IMO, helps make sure people following the steps get stuck at that step, if something is wrong.

Reply:
One issue with the `--wait` is that it doesn't work on my Mac with kind, because LB external IP is pending so the cmd appears hang while the pod is already up.

```
 ~/Downloads/istio-1.21.0/ k get svc -n istio-ingress2
NAME            TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)                                      AGE
istio-ingress   LoadBalancer   10.96.181.157   <pending>     15021:30903/TCP,80:30580/TCP,443:31328/TCP   2m58s
```

Reply:
That makes sense, but the ingress isn't much use without that IP anyway.

We should probably either remove the ingress installation, or specifically mention that you need something like metallb in `kind` if you want an ingress, otherwise we should recommend people to skip the ingress install.

Reply:
> That makes sense, but the ingress isn't much use without that IP anyway.

I'd agree with that in production, but in testing, it is still useful as I can hit istio-ingressgateway svc and perform all types of testing as we documented in our get started guide.